centreon/www/install/php/Update-23.10.0.php

```diff
@@ -353,10 +353,10 @@
                 <<<'SQL'
                     INSERT INTO `dashboard_widgets` (`name`, `version`)
                     VALUES
-                        ('centreon-widget-generictext', '23.10.0'),
-                        ('centreon-widget-singlemetric', '23.10.0'),
-                        ('centreon-widget-graph', '23.10.0'),
-                        ('centreon-widget-topbottom', '23.10.0')
+                        ('centreon-widget-generictext', '23.10.1'),
+                        ('centreon-widget-singlemetric', '23.10.1'),
+                        ('centreon-widget-graph', '23.10.1'),
+                        ('centreon-widget-topbottom', '23.10.1')
```

Comment:
Not sure the version should be bump as we are in the 23.10.0 script

Reply:
you are correct, I will revert these to 23.10.0, to have a clean state for the next release week.